data/types/bolus/prescriptor/prescriptor_test.go

```diff
@@ -0,0 +1,105 @@
+package prescriptor_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	dataNormalizer "github.com/tidepool-org/platform/data/normalizer"
+	"github.com/tidepool-org/platform/data/types/bolus/prescriptor"
+	dataTypesTest "github.com/tidepool-org/platform/data/types/test"
+	errorsTest "github.com/tidepool-org/platform/errors/test"
+	"github.com/tidepool-org/platform/pointer"
+	"github.com/tidepool-org/platform/structure"
+	structureValidator "github.com/tidepool-org/platform/structure/validator"
+	"github.com/tidepool-org/platform/test"
+)
+
+func NewPrescriptor() *prescriptor.Prescriptor {
+	datum := prescriptor.NewPrescriptor()
+	datum.Prescriptor = pointer.FromString(test.RandomStringFromArray(prescriptor.Presciptors()))
+	return datum
+}
+
+func ClonePrescriptor(datum *prescriptor.Prescriptor) *prescriptor.Prescriptor {
+	if datum == nil {
+		return nil
+	}
+	clone := prescriptor.NewPrescriptor()
+	clone.Prescriptor = pointer.CloneString(datum.Prescriptor)
+	return clone
+}
+
+var _ = Describe("Prescriptor", func() {
+	Context("Object Creation", func() {
+
+		It("Manual Prescriptor is expected", func() {
+			Expect(prescriptor.ManualPrescriptor).To(Equal("manual"))
+		})
+
+		It("Auto Prescriptor is expected", func() {
+			Expect(prescriptor.AutoPrescriptor).To(Equal("auto"))
+		})
+
+		It("Hybrid Prescriptor is expected", func() {
+			Expect(prescriptor.HybridPrescriptor).To(Equal("hybrid"))
+		})
+
+		Context("NewPrescriptor", func() {
+			It("is successful", func() {
+				Expect(prescriptor.NewPrescriptor()).To(Equal(&prescriptor.Prescriptor{}))
+			})
+		})
+
+	})
+	Context("Prescriptor", func() {
+		Context("Parse", func() {
+			// TODO
+		})
+
+		Context("Validate", func() {
+			DescribeTable("validates the datum",
+				func(mutator func(datum *prescriptor.Prescriptor), expectedErrors ...error) {
+					datum := NewPrescriptor()
+					mutator(datum)
+					dataTypesTest.ValidateWithExpectedOrigins(datum, structure.Origins(), expectedErrors...)
+				},
+				Entry("succeeds",
+					func(datum *prescriptor.Prescriptor) {},
+				),
+				Entry("Valid value",
```

Comment:
Maybe add a test with a valid sting too ?

Reply:
that's the default case, "succeeds"
let me just rename the "valid value"